language/rust/language.go

```diff
@@ -0,0 +1,152 @@
+package rust
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	pkgerrors "github.com/pkg/errors"
+
+	"github.com/symflower/eval-dev-quality/language"
+	"github.com/symflower/eval-dev-quality/log"
+	"github.com/symflower/eval-dev-quality/util"
+)
+
+// Language holds a Rust language to evaluate a repository.
+type Language struct{}
+
+func init() {
+	language.Register(&Language{})
+}
+
+var _ language.Language = (*Language)(nil)
+
+// ID returns the unique ID of this language.
+func (l *Language) ID() (id string) {
+	return "rust"
+}
+
+// Name is the prose name of this language.
+func (l *Language) Name() (id string) {
+	return "Rust"
+}
```

Comment:
I guess we should a introduce a common language adapter at some point which implements all the basic methods. Like we have for the languages in the Symflower repo.